pnpm-lock.yaml

```diff
@@ -249,15 +249,15 @@ importers:
       typescript: 4.7.4
       vite: ^3.0.0
     dependencies:
-      '@tanstack/query-sync-storage-persister': link:../../../packages/query-sync-storage-persister
-      '@tanstack/react-query': link:../../../packages/react-query
-      '@tanstack/react-query-devtools': link:../../../packages/react-query-devtools
-      '@tanstack/react-query-persist-client': link:../../../packages/react-query-persist-client
+      '@tanstack/query-sync-storage-persister': 4.26.1
+      '@tanstack/react-query': 4.26.1_biqbaboplfbrettd7655fr4n2y
+      '@tanstack/react-query-devtools': 4.26.1_brdhmlf72zuns3lsk66phyptty
+      '@tanstack/react-query-persist-client': 4.26.1_vohxj3xpfxn57cw32hqo6tozmq
```

Comment:
I'm not sure why there are so many changes in the lock file that don't seem related to the packages I bumped. All I did was run `pnpm install` after bumping. For reference, I'm on pnpm `v7.25.0`.